### DISCUSSION TITLE
Self hosted community edition, is that possible to support generic OpenID Connect.

### DISCUSSION THREAD

**Discussion by WestFarmer** [hoppscotch/hoppscotch#3384] [Q&A] [answered]
We are hosting hoppscotch in offline environment

so we can't use github/google/microsoft

can we support Keycloak ?

**Comment by fbuchmeier-abi** [comment, marked as answer]
If you are interested in a preview, I recommend checking out the following PR: https://github.com/hoppscotch/hoppscotch/pull/3266

OIDC integration worked for me with this.
